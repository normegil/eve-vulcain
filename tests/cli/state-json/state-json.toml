bin.name = "eve-vulcain"
args = "state -vvv --json"
status.code = 0
