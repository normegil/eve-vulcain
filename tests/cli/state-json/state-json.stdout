{"character_name":"EveVulcain Test User","isk":123456789.12,"location":{"Facility":{"name":"Maurasi - BPs Center, read BIO","system":{"name":"Maurasi","security_status":0.9127476215362548,"constellation":{"name":"Kimotoro","region":{"name":"The Forge"}}}}},"corporation":{"name":"School of Applied Knowledge","alliance":null},"jobs":{"jobs":[{"item_name":"Drone Link Augmentor I","runs":56,"duration_left":[..]}]},"orders":{"orders":[{"item_name":"Antipharmakon Aeolis","order_type":"Buy","price":20.0,"volume_remain":1,"volume_total":1}]}}
